### ISSUE TITLE
"Single user" visibility setting

### ISSUE THREAD

**Issue by KatharineV** [scientist-softserv/adventist_knapsack#347]
We would like a visibility setting that is a variation on "institutional" visibility. The setting might be called "Institutional - Single User." It would restrict a work to access only by users who have logged in and/or are accessing via an institutional IP address (depending on how the institutional setting works ultimately). Additionally, only one credentialed user could access the work at a time. Perhaps there would be a time limit for viewing, so a single user couldn't leave a work open on their page indefinitely, locking all other users out. Administrators should still be able to view the work. Administrators should be able to recall the work, if there is no time out option. If possible, other users should be able to enter a waiting list to be the next to use the work.

Basically, our use case is that a publisher is willing to let us share a highly valuable and in-demand set of Seventh-day Adventist Bible Commentaries on a Hyku-based repository, but they have specific requirements that include institutional log-in access and single-user access.
